statute.md

```diff
@@ -10,101 +10,126 @@ title: Berlin TWC Statute
 
 2.3
 
-Every proposal must be submitted to the Coordination Team at least 3 days before the Assembly. A proposal should be in the form of a binding statement that can be either approved or rejected. If the statement binds a specific Team, it must be said explicitly. 
+Every proposal must be submitted to the Coordination team at least 3 days before the assembly. A proposal should be in the form of a binding statement that can be either approved or rejected. If the statement binds a specific team, it must be said explicitly. 
 
 ### 3
 
-A Team is a self-organised group of Berlin TWC members. A member can belong to zero or more teams. Every Team must have at least one Credentials' holder (defined later) for the tools necessary for the team to perform their tasks.
+A team is a self-organised group of Berlin TWC members. A member can belong to zero or more teams. Every team must have at least one credentials holder (defined later) with the tools necessary for the team to perform their tasks.
 
-Every Team must guarantee total transparency to the Assembly for any matter that impact Berlin TWC. Failure in doing so, once acknowledged by the Assembly, will automatically trigger a disband vote.
+Every team must guarantee total transparency to the assembly for any matter that impacts Berlin TWC. Failure to do so, once acknowledged by the assembly, will automatically trigger a disband vote.
 
-The Assembly respects the organizational autonomy of a Team as long as no other Team or Berlin TWC is impacted.
+The assembly respects the organisational autonomy of a team as long as no other team or Berlin TWC is impacted.
 
-Participation in a group of a member is decided by the group itself according to their internal necessities and preferences.
+Participation in a team of a member is decided by the group itself according to their internal needs and preferences.
 
 3.1 
 
-A Team is created in an Assembly through vote. The responsibilities and capabilities of the Team must be clearly specified. These can be expanded or restricted through an Assembly Vote. The creation of a Team can be specified in the Agenda as a self-standing item or as part of a wider proposal. The vote for the creation of a Team can be held without inserting it into any Agenda if the discussion of a proposal suggests it as a necessity. The proposal for the creation of a Team must include its initial members.
+A team is created in an assembly through vote. The responsibilities and capabilities of the team must be clearly specified. These can be expanded or restricted through an assembly vote. The creation of a team can be specified in the Agenda as a self-standing item or as part of a wider proposal. The vote for the creation of a team can be held without inserting it into any Agenda if the discussion of a proposal suggests it as a necessity. The proposal for the creation of a team must include its initial members.
```

Comment:
So according to this, we could vote for a steering committee, for a 6 month duration period without any constitutional amendments? The past months there have been steering committee shadow meetings, and I would like to formalize who is in the steering committee, especially as we interface with many external organizations  who want to jointly host events with us etc...

Reply:
I would say so. It's a special team with a lot of power, but I think we don't need to change this part. I would add it to the list of special teams though.